tests/test_dynamics_start_date.py

```diff
@@ -98,3 +98,18 @@ def test_run(self):
                     # Verify by primary key the same records are replicated in the 1st and 2nd syncs
                     self.assertTrue(primary_keys_sync_1.issubset(primary_keys_sync_2),
                                     msg="Records in the 1st sync are not a subset of the 2nd sync")
+
+                elif expected_replication_method == self.FULL_TABLE:
+                    # Verify that the 2nd sync with an earlier start date replicates a greater or equal number of
+                    # records as the 1st sync.
+                    self.assertGreaterEqual(record_count_sync_2, record_count_sync_1,
```

Comment:
This should be just greater or just equal depending on the implementation of the start date in Full Table streams.

Reply:
The API doesn't have a way to filter full table streams, so I've adjusted this to be just equal.